pxr/usdImaging/usdImaging/textureUtils.cpp

```diff
@@ -58,7 +58,7 @@ UsdImaging_GetUdimTiles(
                 layerHandle, TfStringPrintf(formatString.c_str(), t))
             : TfStringPrintf(formatString.c_str(), t);
         if (!resolver.Resolve(path).empty()) {
-            ret.emplace_back(t - startTile, TfToken(path));
+            ret.emplace_back(t - startTile, TfToken(resolver.Resolve(path)));
```

Comment:
I've called `Resolve` again here as resolution happens in the context of a resolver scope cache.

https://github.com/PixarAnimationStudios/USD/blob/0f4dfeee951f32934f08d4e6ee8efcd9cfda2f68/pxr/usdImaging/usdImaging/textureUtils.cpp#L47

Reply:
Hi @fabal - thanks for this!  As it turns out, we are in the midst of a complete rewrite of Storm's texture manager, and in changes that I believe will be pushed out to the github dev branch shortly, we have also "resolved" this UDIM issue as part of that work!